src/test/pipenv-test-app/2.7/pipfile.tpl

```diff
@@ -0,0 +1,96 @@
+{% macro content() %}
+{
```

Comment:
Thanks! That's much better than having it in one file. However, would it be possible to solve without having to modify the individual `Pipfile.lock` files by adding this macro definition on top? In an ideal situation, one would just copy  the `Pipfile.lock` file into the proper location, overwrite what's there and that'd be it. No modifying of the contents of the file.

Perhaps the `include` directive can be used in the `src/test/pipenv-test-app/Pipfile.lock` file?

    {% include 'src/test/pipenv-test-app/" + spec.version + "/Pipfile.lock' %}

Reply:
This also works